pallets/pallet-migration/src/lib.rs

```diff
@@ -228,7 +228,7 @@ pub mod pallet {
 				if !is_migrated {
 					let key_hash = <T as frame_system::Config>::Hashing::hash(&storage_key);
 					MigratedKeys::<T>::insert(key_hash, ());
-					pallet_web3_names::migrations::update_balance_for_w3n::<T>(key)
+					pallet_web3_names::migrations::update_balance_for_w3n::<T, _>(key)
```

Comment:
It's not important actually, since we are only migrating our currently deployed instance, which has a `()` generic. So I removed the second generic from that function: https://github.com/KILTprotocol/kilt-node/pull/779/commits/d5703c9e0b9c41d824571958a4eb353ef29721f7.

Reply:
So, still good catch! 😄

Reply:
Yeah, I was confused about how it would work. Only the existing pallet needs the migration, but if a blanket is provided, how does the implementation know which instance should be executed. But now It's pretty clear - at least for me